<launch>
    <arg name="gui" default="false" />
    <arg name="mon-output" default="screen" />

    <node pkg="monitoring_visualization" type="gui_msg_concat_node" name="gui_concat" respawn="true" output="$(arg mon-output)" if="$(arg gui)" />
    <node pkg="monitoring_monitors_ros" type="monitorstatistics.py" name="statistics_monitor" output="$(arg mon-output)" respawn="true">
      <rosparam command="load" file="$(find monitoring_monitors_ros)/config/statisticstestconfig.yaml" />
    </node>

</launch>
